src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSCommand.groovy

```diff
@@ -224,12 +179,12 @@ class PBSCommand extends Command {
     String assembleDependencyString() {
         StringBuilder qsubCall = new StringBuilder("")
         LinkedList<String> tempDependencies =
-                creatingJob.getParentJobIDsAsString().findAll {
-                    it != "" && it != NONE && it != "-1"
+                creatingJob.getParentJobIDs().findAll {
+                    it.getId() != "" && it.getId() != NONE && it.getId() != "-1"
                 }.collect {
-                    it.split("\\.")[0] // Keep the command line short. PBS accepts the job number for dependencies.
+                    it.getId().split("\\.")[0] // Keep the command line short. PBS accepts the job number for dependencies.
                 } as LinkedList<String>
-        if (creatingJob.getParentJobIDsAsString().any { it.contains("[].") }) {
+        if (creatingJob.getParentJobIDs().any { it.getId().contains("[].") }) {
```

Comment:
I don't really get the difference between any and find. Especially, when it comes to closures. 
.find?

Reply:
I think this code is not needed anyway, because it's for array jobs